lib/invoca/utils/enumerable.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Invoca ::Enumerable extensions
+module Enumerable
+  def build_hash(res = {})
+    each do |x|
+      pair = block_given? ? yield(x) : x
+      res[pair.first] = pair.last if pair
+    end
+    res
+  end
+
+  class MultiSender
+
+    undef_method(*(instance_methods.map { |m| m.to_s } - %w*__id__ __send__ object_id*))
```

Comment:
Gah, that's crazy they used `%w*`! What were they thinking? `%w[ ... ]` makes it so much more clear that it's a (space-separated) array:
```
%w[__id__ __send__ object_id]
```
But even better: starting with Ruby 1.9, the instance methods are symbols, we can avoid the strings and `%w` entirely:
```
undef_method(*(instance_methods - [:__id__, :__send__, :object_id]))
```

Reply:
Yeah, I didn't even want to touch that line when I saw it. Wasn't worth research time to fix it. But, hey, if you know how to do it off the top of your head, why not!